CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.3.0 (2020-04-27)
+
+- Added `NavigationExtras.state` support to the flyout component's permalink button. [#78](https://github.com/blackbaud/skyux-flyout/pull/78) (Thanks [@Alex-Vaky](https://github.com/Alex-Vaky)!)
```

Comment:
Since `NavigationExtras.state` is a code snippet, I assumed that we would have that exactly in the code in the linked PR, but we don't. We do have `[state]="permalink.route.extras?.state"` though. Is `NavigationExtras.state` correct?

Reply:
Yes, that's the underlying mechanism that `[state]` controls. `NavigationExtras.state` is also in Angular's documentation for `[state]`.

Reply:
Great. I was hoping it was just be an Angular thing I was unfamiliar with.